tests/remote/test_thunderbird.py

```diff
@@ -88,9 +88,9 @@ def test_daily_scraper(tmpdir, args):
     {'application': 'thunderbird', 'branch': 'comm-central', 'platform': 'win32', 'locale': 'de'},
     {'application': 'thunderbird', 'branch': 'comm-central', 'platform': 'win32', 'extension': 'txt'},
 ])
+@pytest.mark.xfail(run=False, reasson="tinderbox builds not available in the archive")
```

Comment:
I would still prefer the `strict` argument here, given that once we have the missing feature implemented the test will unexpectedly pass. Otherwise we might miss to re-enable the test. Also please fix `s/reasson/reason/`